misc/scripts/upgrade_test.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+    MANAGED_PLUGIN_NAME="vsphere:latest"
+    E2E_Tests="github.com/vmware/docker-volume-vsphere/tests/e2e"
+    GO="go"
+    SSH="ssh -i /root/.ssh/id_rsa -kTax -o StrictHostKeyChecking=no"
+
+    get_vib_url() {
+        echo "Get version $1"
```

Comment:
It will be a pain to keep adding `elif` block .. better way to do is introducing a map OR a mapping text file (`"manage the mapping into test code or text file which contains (version_tag : vib_url where version_tag refers known version number and vib_url refers the download link for the VIB). It will be easier to validate the user input as well."`)

reference: https://github.com/vmware/docker-volume-vsphere/issues/1380#issuecomment-313266842